install/readme.md

```diff
@@ -0,0 +1,80 @@
+# SODA Experience Installer
+
+The SODA Experience installer enables users to install the SODA Authentication module, Keystone, and the SODA Dashboard independently.  
+
+
+## Pre-requisites
+
+- This installer is currently tested and works on **Ubuntu 20.04**. 
+
+- Python  >= 3.8.10 ***(This installer may not work on Ubuntu 18.04 since the python3 version may not be supported.)***
```

Comment:
Please check delfin PR #942 which works for the 18.04. (You can try to use ministone.py from that PR that contain some fixes)

Reply:
@joseph-v PR #942 in the installer repository?

Reply:
I used the ministone.py from that PR and tried on a Ubuntu 19=8.04 setup. I am getting the following exception:
```
Execption: HTTPConnectionPool(host='192.168.56.1', port=80): Max retries exceeded with url: /identity/v3/auth/tokens (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fd0243a5588>: Failed to establish a new connection: [Errno 111] Connection refused',))
```

But the same script worked in Ubuntu 20.04.

I checked online and there seem to be some dependencies that may be deprecated. 
![soda_experience_install_python3 6_deprecated](https://user-images.githubusercontent.com/19162717/203245065-f5c355dd-aa7d-428b-bace-a841bb3fa3ac.png)